tables/tr.ctb

```diff
@@ -82,32 +104,222 @@ lowercase m 134		LATIN SMALL LETTER M
+base uppercase        \x014E \x014F      Ŏŏ  &#334;&#335;   LATIN LETTER O WITH BREVE
+base uppercase        \x0150 \x0151      Őő  &#336;&#337;   LATIN LETTER O WITH DOUBLE ACUTE
+base uppercase        \x0154 \x0155      Ŕŕ  &#340;&#341;   LATIN LETTER R WITH ACUTE
+base uppercase        \x0156 \x0157      Ŗŗ  &#342;&#343;   LATIN LETTER R WITH CEDILLA
+base uppercase        \x0158 \x0159      Řř  &#344;&#345;   LATIN LETTER R WITH CARON
+base uppercase        \x015A \x015B      Śś  &#346;&#347;   LATIN LETTER S WITH ACUTE
+base uppercase        \x015C \x015D      Ŝŝ  &#348;&#349;   LATIN LETTER S WITH CIRCUMFLEX
+base uppercase        \x0160 \x0161      Šš  &#352;&#353;   LATIN LETTER S WITH CARON
+base uppercase        \x0218 \x0219      Șș                 LATIN LETTER S WITH COMMA BELOW
+base uppercase        \x1e9e \x00DF      ß                  LATIN SMALL LETTER SHARP S
+base uppercase        \x0162 \x0163      Ţţ  &#354;&#355;   LATIN LETTER T WITH CEDILLA
+base uppercase        \x0164 \x0165      Ťť  &#356;&#357;   LATIN LETTER T WITH CARON
+base uppercase        \x0166 \x0167      Ŧŧ  &#358;&#359;   LATIN LETTER T WITH STROKE
+base uppercase        \x0191 \x0192      Ƒƒ  &#401;&#402;   LATIN LETTER F WITH HOOK
+base uppercase        \x021A \x021B      Țț                 LATIN LETTER T WITH COMMA BELOW
+base uppercase        \x0168 \x0169      Ũũ  &#360;&#361;   LATIN LETTER U WITH TILDE
+base uppercase        \x016A \x016B      Ūū  &#362;&#363;   LATIN LETTER U WITH MACRON
+base uppercase        \x016C \x016D      Ŭŭ  &#364;&#365;   LATIN LETTER U WITH BREVE
+base uppercase        \x016E \x016F      Ůů  &#366;&#367;   LATIN LETTER U WITH RING ABOVE
+base uppercase        \x0170 \x0171      Űű  &#368;&#369;   LATIN LETTER U WITH DOUBLE ACUTE
+base uppercase        \x0172 \x0173      Ųų  &#370;&#371;   LATIN LETTER U WITH OGONEK
+base uppercase        \x0174 \x0175      Ŵŵ  &#372;&#373;   LATIN LETTER W WITH CIRCUMFLEX
+base uppercase        \x0176 \x0177      Ŷŷ  &#374;&#375;   LATIN LETTER Y WITH CIRCUMFLEX
+base uppercase        \x0179 \x017A      Źź  &#377;&#378;   LATIN LETTER Z WITH ACUTE
+base uppercase        \x017B \x017C      Żż  &#379;&#380;   LATIN LETTER Z WITH DOT ABOVE
+base uppercase        \x017D \x017E      Žž  &#381;&#382;   LATIN LETTER Z WITH CARON
+base uppercase        \x01CD \x01CE      Ǎǎ  &#461;&#462;   LATIN LETTER A WITH CARON
+base uppercase        \x1E80 \x1E81      Ẁẁ  &#7808;&#7809; LATIN LETTER W WITH GRAVE
+base uppercase        \x1E82 \x1E83      Ẃẃ  &#7810;&#7811; LATIN LETTER W WITH ACUTE
+base uppercase        \x1E84 \x1E85      Ẅẅ  &#7812;&#7813; LATIN LETTER W WITH DIAERESIS
+base uppercase        \x1EF2 \x1EF3      Ỳỳ  &#7922;&#7923; LATIN LETTER Y WITH GRAVE
+
+# Correct order of comma and numeric indicator
+match %a , %# 2-3456
+noback pass2 @3456-2-3456 @2-3456
+
+# Correct order of dot and numeric indicator
+match %a . %# 256-345
+noback pass2 @3456-256-3456 @256-3456
```

Comment:
There is no numsign in this table, so why are these rules needed?

Reply:
I just deleted the rules, I don't think we need them. We'll notice if we do anyway.